.ci/configure_crio_for_kata.sh

```diff
@@ -16,9 +16,13 @@ crio_config_dir="/etc/crio/crio.conf.d"
 runc_flag="\/usr\/local\/bin\/crio-runc"
 kata_flag="\/usr\/local\/bin\/containerd-shim-kata-v2"
 
+echo "crio --version!!!!!!!!!!!!"
```

Comment:
I think you missed an exclamation mark there! :smile: But seriously, all the exclamation marks might cause you problems due to the special meaning of `!!`.

Reply:
Sorry, this is only a test :-)